tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -121,6 +121,18 @@ def setup_method(self):
         resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Precondition: Verified cluster is up and running and all pods are online.")
+        LOGGER.info("Get %s pod to be deleted", const.POD_NAME_PREFIX)
```

Comment:
Why this change ?
And if it's common, can we add lib for this, as we'll need these code in most of the files.

Reply:
We are adding this for now in setup method untill we get another way for pod deletion. Once we have another method we can add a wrapper lib to choose a method but till then we can keep this as is.